client/web/compose/src/components/PageBlocks/Navigation/Configurator.vue

```diff
@@ -0,0 +1,291 @@
+                      <font-awesome-icon
+                        :icon="['fas', 'bars']"
+                        class="grab text-grey"
+                      />
+                    </td>
+                    <td class="align-middle">
+                      <b-form-select
+                        v-model="column.type"
+                        :options="navigationItemTypes"
+                        size="sm"
+                      >
+                        <template #first>
+                          <b-form-select-option :value="''">
+                            {{ $t("navigation.none") }}
+                          </b-form-select-option>
+                        </template>
+                      </b-form-select>
+                    </td>
+                    <td class="align-middle">
+                      <b-form-input
+                        v-model="column.options.textColor"
+                        size="sm"
+                        type="color"
+                        debounce="300"
+                        class="w-100"
+                      />
+                    </td>
+                    <td class="align-middle">
+                      <b-form-select
+                        v-model="column.options.backgroundColor"
+                        :options="backgroundColors"
+                        size="sm"
+                        type="color"
+                        class="w-100"
+                      >
+                        <template #first>
+                          <b-form-select-option :value="''">
+                            {{ $t("navigation.none") }}
+                          </b-form-select-option>
+                        </template>
+                      </b-form-select>
+                    </td>
+                    <td class="align-middle text-center">
+                      <b-form-checkbox
+                        v-model="column.options.enabled"
+                        name="enabled-button"
+                        switch
+                        class="mb-0"
+                      />
+                    </td>
+                    <td class="align-middle">
+                      <c-input-confirm
+                        size="lg"
+                        @confirmed="options.lists.splice(index, 1)"
+                      />
+                    </td>
+                  </tr>
+
+                  <component
+                    :is="column.type"
+                    :column="column"
+                    :namespace="namespace"
+                    @update="(value) => column = value"
+                  />
+                  <Text />
+                </tbody>
+              </b-table-simple>
+            </div>
+          </draggable>
+        </div>
+      </div>
+    </b-tab>
+  </div>
+</template>
+
+<script>
+import base from '../base'
+import Draggable from 'vuedraggable'
+import { compose } from '@cortezaproject/corteza-js'
+import TextSection from './NavTypes/Text.vue'
```

Comment:
Name consistency with other types, call it Text

Reply:
because there seem to be an existing html tag named `<text>` vue throws an error in the console because the component and the html tag has the same name and it won't render

Reply:
You can still import it as text and just define it with text-section for example